src/default_snippets.js

```diff
@@ -50,6 +50,7 @@
 	{trigger: "ee", replacement: "e^{ $0 }$1", options: "mA"},
     {trigger: "invs", replacement: "^{-1}", options: "mA"},
     {trigger: /([A-Za-z])(\d)/, replacement: "[[0]]_{[[1]]}", options: "rmA", description: "Auto letter subscript", priority: -1},
+	{trigger: "\\\\(${MORE_SYMBOLS}|${SYMBOL})(.)", replacement: "\\[[0]] [[1]]", options: "rmA", description: "Manage symbols exceptions"},
```

Comment:
Do we need to capture all characters here? Can we apply the fix only to numbers typed after a symbol?

Also, do we want make "log" an exception to the rule (i.e. continue to apply auto-subscript to it), so we can type e.g. `log2` -> `\log_{2}`?

Reply:
Like you want. I done a new commit for this exception.